apps/probate/probate-cron-hmrc-extract/probate-cron-hmrc-extract.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: probate-cron-hmrc-extract
+spec:
+  releaseName: probate-cron-hmrc-extract
+  values:
+    job:
+      environment:
+        TASK_NAME: HmrcExtractTask
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/probate/back-office:pr-2079-cdf3cd3-20221028130320 #{"$imagepolicy": "flux-system:demo-probate-back-office"}
```

Comment:
Don't have a demo image in the base config, this is why CI check is failing, either remove reference or update to prod image

Reply:
Removed